docs/src/free_associative_algebra.md

```diff
@@ -33,7 +33,7 @@ characteristic(R::FreeAssAlgebra)
 ## Free algebra constructors
 
 ```julia
-FreeAssociativeAlgebra(R::Ring, s::Vector{<: Union{String, Char, Symbol}}; cached::Bool = true)
+FreeAssociativeAlgebra(R::Ring, s::Union{Vector{String}, Vector{Char}, Vector{Symbol}}; cached::Bool = true)
```

Comment:
Would AbstractVector{T} be better here? And where T <: Union{AbstractString, Char, Symbol}?

Reply:
My goal was to reuse code while disallowing vectors that mix the types, such as `Vector{Union{Char, String, Symbol}}`. Or, is there no need to require homogeneous vectors for the variables?

Reply:
I am going with the union of abstract vectors.